assets/sass/authorize-application/_googlesitekit-authorize-application.scss

```
@@ -19,3 +19,7 @@
 .auth-app-card #approve {
 	background-color: $c-deep-blue;
 }
+
+#wpfooter #footer-thankyou a {
```

Hmm, actually seeing as element IDs are unique on a page, we can simplify this selector. I'll make the change myself to save the round trip.

```suggestion
#footer-thankyou a {
```